Rubberduck.CodeAnalysis/Inspections/Abstract/QuickFixBase.cs

```diff
@@ -39,7 +40,9 @@ public void RemoveInspections(params Type[] inspections)
             _supportedInspections = _supportedInspections.Except(inspections).ToHashSet();
         }
 
-        public abstract void Fix(IInspectionResult result);
+        public virtual CodeKind TargetCodeKind => CodeKind.CodePaneCode;
+
+        public abstract void Fix(IInspectionResult result, IRewriteSession rewriteSession = null);
```

Comment:
Is the default of `null` assignment temporary as it's WIP? I would prefer in the final version that it wasn't?

Reply:
That is just temporary. I did want to put this PR out before changing a gazillion unit tests.